src/TAlias.cpp

```diff
@@ -116,24 +118,15 @@ bool TAlias::match(const QString& toMatch)
     }
 
     if (rc < 0) {
-        switch (rc) {
-        case PCRE_ERROR_NOMATCH:
-            goto MUD_ERROR;
-
-        default:
-            goto MUD_ERROR;
-        }
-    }
-    if (rc > 0) {
+        goto MUD_ERROR;
```

Comment:
Did you really code in a `goto` here :stuck_out_tongue_closed_eyes: ?? (Disclaimer: I have barely any C++ knowledge, so I back out again)

Reply:
Just refactoring the _existing_ code which used it - but yeah I did want to put on a pair of rubber gloves (good for protection against electric shocks BTW) before hand... :nauseated_face: